Error: Cannot find module '@babel/compat-data/corejs3-shipped-proposals'

<!--
    Before posting this issue, please perform the following steps, and confirm that the issue is still present.

1. Update Laravel Mix to the latest version. (`npm update laravel-mix` or `yarn upgrade laravel-mix --latest`)
2. Nuke your dependencies, and reinstall from scratch: `rm -rf node_modules && npm cache clear && npm install`.
3. Check your `package.json` file, and ensure that there are no old Laravel Elixir dependencies that might be interfering with Mix.
-->

-   Laravel Mix Version: 5.0.4 
-   Node Version: 13.13.0
-   NPM Version: 6.14.4
-   OS: macOS 10.15.4

I think this is related to a breaking change in node version 13.13 which conflicts with babel <= 7.9

I am still facing this issue in an ember application, after multiple attempts to upgrade all packages or completely rm-ing+DL-ing `node_modules` and `yarn clean cache`.

Error
```
Build Error (broccoli-persistent-filter:Babel > [Babel: ember-cookies]) in ember-cookies/clear-all-cookies.js

Cannot find module '@babel/compat-data/corejs3-shipped-proposals'
Require stack:
- <REDACTED>/node_modules/@babel/preset-env/lib/polyfills/corejs3/usage-plugin.js
- <REDACTED>/node_modules/@babel/preset-env/lib/index.js
- <REDACTED>/node_modules/@babel/core/lib/config/files/plugins.js
- <REDACTED>/node_modules/@babel/core/lib/config/files/index.js
- <REDACTED>/node_modules/@babel/core/lib/index.js
- <REDACTED>/node_modules/broccoli-babel-transpiler/lib/worker.js


Stack Trace and Error Report: /tmp/error.dump.1ba436bec65572a52ffe5a2a012c4790.log
```

`package.json`
```json
{
  "private": true,
  "repository": "",
  "author": "",
  "directories": {
    "doc": "doc",
    "test": "tests"
  },
  "main": "app.js",
  "scripts": {
    "build": "ember build",
    "lint:hbs": "ember-template-lint .",
    "lint:js": "eslint .",
    "start": "ember serve",
    "test": "ember test",
    "publish": "ember build --environment=production",
    "postpublish": "yarn bestzip dist.zip .ebextensions dist/"
  },
  "devDependencies": {
    "@ember/optional-features": "^1.3.0",
    "@fortawesome/ember-fontawesome": "^0.2.1",
    "@fortawesome/free-brands-svg-icons": "^5.13.0",
    "@fortawesome/free-solid-svg-icons": "^5.13.0",
    "bestzip": "^2.1.5",
    "bootstrap": "^4.5.0",
    "broccoli-asset-rev": "^3.0.0",
    "ember-auto-import": "^1.5.3",
    "ember-bootstrap": "^3.1.3",
    "ember-cli-app-version": "^3.2.0",
    "ember-cli-babel": "7.19.0",
    "ember-cli-dependency-checker": "^3.2.0",
    "ember-cli-deploy": "^1.0.2",
    "ember-cli-deploy-build": "^2.0.0",
    "ember-cli-deploy-compress": "^0.3.2",
    "ember-cli-deploy-gzip": "^1.0.1",
    "ember-cli-eslint": "^5.1.0",
    "ember-cli-htmlbars": "^4.3.1",
    "ember-cli-inject-live-reload": "^2.0.2",
    "ember-cli-sass": "^10.0.1",
    "ember-cli-sri": "^2.1.1",
    "ember-cli-template-lint": "^2.0.0",
    "ember-cli-uglify": "^3.0.0",
    "ember-concurrency": "^1.1.7",
    "ember-cookies": "^0.5.2",
    "ember-export-application-global": "^2.0.1",
    "ember-fetch": "^8.0.1",
    "ember-g-recaptcha": "^1.3.0",
    "ember-intl": "^4.3.1",
    "ember-load-initializers": "^2.1.1",
    "ember-local-storage": "^1.7.2",
    "ember-metrics": "^0.14.0",
    "ember-qunit": "^4.6.0",
    "ember-resolver": "^8.0.0",
    "ember-router-scroll": "^3.3.2",
    "ember-search-helper": "^2.0.0",
    "ember-source": "^3.18.1",
    "ember-truth-helpers": "^2.1.0",
    "eslint": "^7.0.0",
    "eslint-plugin-ember": "^8.5.1",
    "loader.js": "^4.7.0",
    "qunit-dom": "^1.2.0",
    "sass": "^1.26.5",
    "sitemap-autogenerator": "^1.2.3"
  },
  "engines": {
    "node": ">= 10.*"
  },
  "dependencies": {
    "@glimmer/component": "^1.0.0",
    "@glimmer/tracking": "^1.0.0",
    "big.js": "^5.2.2",
    "dayjs": "^1.8.26",
    "ember": "^1.0.3",
    "ember-anchor": "^1.0.3",
    "ember-cli": "^3.18.0",
    "ember-useragent": "^0.9.1",
    "query-string": "^6.12.1"
  }
}

--- a/package.json
+++ b/package.json
@@ -31,12 +31,12 @@
         "src"
     ],
     "dependencies": {
-        "@babel/core": "^7.2.0",
-        "@babel/plugin-proposal-object-rest-spread": "^7.2.0",
-        "@babel/plugin-syntax-dynamic-import": "^7.2.0",
-        "@babel/plugin-transform-runtime": "^7.2.0",
-        "@babel/preset-env": "^7.2.0",
-        "@babel/runtime": "^7.2.0",
+        "@babel/core": "^7.9.0",
+        "@babel/plugin-proposal-object-rest-spread": "^7.9.0",
+        "@babel/plugin-syntax-dynamic-import": "^7.8.0",
+        "@babel/plugin-transform-runtime": "^7.9.0",
+        "@babel/preset-env": "^7.9.0",
+        "@babel/runtime": "^7.9.0",
         "autoprefixer": "^9.4.2",
         "babel-loader": "^8.0.4",
         "babel-merge": "^2.0.1",

--- a/test/features/vue.js
+++ b/test/features/vue.js
@@ -19,7 +19,6 @@ test.serial.cb('it appends vue styles to your sass compiled file', t => {
   color: red;
 }
 
-
 .hello {
   color: blue;
 }`;
@@ -49,7 +48,6 @@ test.serial.cb('it prepends vue styles to your less compiled file', t => {
         let expected = `body {
   color: pink;
 }
-
 .hello {
   color: blue;
 }`;
@@ -75,8 +73,7 @@ test.serial.cb(
                 File.exists('test/fixtures/fake-app/public/css/vue-styles.css')
             );
 
-            let expected = `
-.hello {
+            let expected = `.hello {
   color: blue;
 }`;
 
@@ -121,8 +118,7 @@ test.serial.cb('it extracts vue Stylus styles to a dedicated file', t => {
     compile(t, config => {
         t.true(File.exists('test/fixtures/fake-app/public/css/components.css'));
 
-        let expected = `
-.hello {
+        let expected = `.hello {
   margin: 10px;
 }
 `;
@@ -174,8 +170,7 @@ test.serial.cb('it extracts vue .scss styles to a dedicated file', t => {
             File.find('test/fixtures/fake-app/public/css/app.css').read()
         );
 
-        expected = `
-.hello {
+        expected = `.hello {
   color: blue;
 }`;
 
@@ -213,8 +208,7 @@ test.serial.cb('it extracts vue .sass styles to a dedicated file', t => {
             File.find('test/fixtures/fake-app/public/css/app.css').read()
         );
 
-        expected = `
-.hello {
+        expected = `.hello {
   color: black;
 }`;
 
@@ -259,8 +253,7 @@ test.serial.cb('it extracts vue Less styles to a dedicated file', t => {
     compile(t, config => {
         t.true(File.exists('test/fixtures/fake-app/public/css/components.css'));
 
-        let expected = `
-.hello {
+        let expected = `.hello {
   color: blue;
 }
 `;

